build.assets/keychain-setup.sh

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
```

Comment:
I do this when things need to work on a Mac because often there is a newer version of bash in /usr/local/bin because the system one is so terribly old. But I see your point and I have been making sure not to use any bash 4+ features so yeah, I'll just use /bin/bash.

To be honest though, I don't think a malicious bash is an issue. Every command run from the script is found via the path, so any of them could be malicious too. Or is there some sort of other attack that I'm missing?

Reply:
This is mostly habit/one less thing to worry about. In other contexts I wouldn't take anything from path either, but that was somewhere else.

I thought you'd mention macOS bash. Yep, it's awful.